dev_guide/builds.adoc

```diff
@@ -1362,9 +1366,19 @@ git:
     name: "<committer name>"
     email: "<committer e-mail>"
   message: "<commit message>"
+env:
+   - name: "<variable name>"
+     value: "<variable value>"
+
 ----
 ====
 
+Similar to the link:#buildconfig-environment[*BuildConfig* environment] variables, the environment variables
+defined in the above payload will be made available to your build.  When variable names in the above payload
+collide with the names of the link:#buildconfig-environment[*BuildConfig* environment] variables, the variables
+in the payload will take precedence.  By default, environment variables passed via webhook are ignored.
+You must set allowEnv to true on the webhook definition to enable this behavior.
```

Comment:
On Tue, Apr 26, 2016 at 5:36 PM, Ben Parees <notifications@github.com>
wrote:

> In dev_guide/builds.adoc
> <https://github.com/openshift/openshift-docs/pull/1893#discussion_r61169059>
> :
>
> >  ----
> >  ====
> >
> > +Similar to the link:#buildconfig-environment[*BuildConfig* environment] variables, the environment variables
> > +defined in the above payload will be made available to your build.  When variable names in the above payload
> > +collide with the names of the link:#buildconfig-environment[*BuildConfig* environment] variables, the variables
> > +in the payload will take precedence.  By default, environment variables passed via webhook are ignored.
> > +You must set allowEnv to true on the webhook definition to enable this behavior.
>
> markup allowEnv with *allowEnv* and lgtm.
>

Done

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/1893/files/3318f76494b7bd078a936466eec298b4df4edcbb#r61169059>
>